### PR TITLE
Publish falcon 7b and 40b

### PR DESCRIPTION
**Pull request by novaturient95**
Falcon model asset migration to public registry

### CODE REVIEW

**Comment by novaturient95** [comment]
Update descriptio

**Comment by choudharykartik** [comment]
Save blob run link: https://ml.azure.com/runs/khaki_endive_t9t3ldn8xp?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

conda.yaml: 
channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.3.1
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.3.1
  - azureml-core==1.52.0
  - azureml-mlflow==1.52.0
  - azureml-metrics==0.0.24
  - scikit-learn==1.1.0
  - cryptography==41.0.2
  - python-dateutil==2.8.2
  - datasets==2.14.4
  - soundfile==0.12.1
  - librosa==0.10.0.post2
  - diffusers==0.19.3
  - sentencepiece==0.1.99
  - transformers==4.31.0
  - torch==2.0.0
  - accelerate==0.21.0
  - Pillow==9.4.0
  - einops
  - azureml-evaluate-mlflow==0.0.24
- pycocotools=2.0.4
name: mlflow-env



MLModel:
flavors:
  hftransformersv2:
    code: null
    config_hf_load_kwargs:
      trust_remote_code: true
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForCausalLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: tiiuae/falcon-40b
    model_data: data
    model_hf_load_args:
      trust_remote_code: true
    pytorch_version: 2.0.0+cu117
    task_type: text-generation
    tokenizer_hf_load_kwargs:
      trust_remote_code: true
    transformers_version: 4.31.0
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.3.1
model_uuid: 9eec31aa11e447e2a3b9ff4a1ca17b33
utc_time_created: '2023-09-22 12:14:01.214117'

File: assets/models/system/tiiuae-falcon-40b/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: tiiuae-falcon-40b
+path: ./
+properties:
+  SHA: c47b371b31a68349c233104050ac76680b8485db
```
Comment (novaturient95) [inline assets/models/system/tiiuae-falcon-40b/spec.yaml:5]: @choudharykartik please help to confirm SHA and update if modified

File: assets/models/system/tiiuae-falcon-7b/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - Standard_NC24rs_v3
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_A100_v4
+  finetune_compute_allow_list:
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_A100_v4
+  model_specific_defaults:
+    apply_deepspeed: 'false'
```
Comment (novaturient95) [inline assets/models/system/tiiuae-falcon-7b/spec.yaml:40]: remove apply_deepspeed. By default its `false`

File: assets/models/system/tiiuae-falcon-7b/spec.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  inference-recommended-sku: Standard_NC12s_v3
+  finetuning-tasks: text-classification, question-answering
+tags:
+  Featured: ''
+  Preview: ''
+  license: apache-2.0
+  task: text-generation
+  author: tiiuae
+  inference_compute_allow_list:
+  - Standard_E8s_v3
```
Comment (novaturient95) [inline assets/models/system/tiiuae-falcon-7b/spec.yaml:18]: @MallikharjunaThota123 please help to confirm allow list for inference

File: assets/models/system/tiiuae-falcon-7b/model.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: huggingface/tiiuae-falcon-7b/2f5c3cd/mlflow_model_folder
```
Comment (novaturient95) [inline assets/models/system/tiiuae-falcon-7b/model.yaml:3]: @MallikharjunaThota123 please update the path once model is saved to blobstorage

File: assets/models/system/tiiuae-falcon-7b/spec.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: tiiuae-falcon-7b
+path: ./
+properties:
+  SHA: 2f5c3cd4eace6be6c0f12981f377fb35e5bf6ee5
```
Comment (novaturient95) [inline assets/models/system/tiiuae-falcon-7b/spec.yaml:5]: @MallikharjunaThota123 update the SHA

File: assets/models/system/tiiuae-falcon-40b/spec.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  datasets: tiiuae/falcon-refinedweb
+  languages: en, de, es, fr
+  inference-min-sku-spec: 24|4|448|1344
+  inference-recommended-sku: Standard_ND40rs_v2
```
Comment (novaturient95) [inline assets/models/system/tiiuae-falcon-40b/spec.yaml:9]: @choudharykartik  add and reconfirm evaluation, finetuning, inference compute allowlist and recommended SKUs

Comment (choudharykartik) [reply assets/models/system/tiiuae-falcon-40b/spec.yaml:9]: Updated

File: assets/models/system/tiiuae-falcon-40b/spec.yaml (hunk 1 above)
Comment (choudharykartik) [reply assets/models/system/tiiuae-falcon-40b/spec.yaml]: updated

**Comment by novaturient95** [comment]
Closing as PRs merged by Kartik, Mallikharjun

**Pull request closed without merging**